sw/host/opentitanlib/src/app/config/opentitan_cw310.json

```diff
@@ -1,36 +1,69 @@
 {
   "includes": ["/__builtin__/opentitan.json"],
   "interface": "cw310",
+  // Below we define the pins and its default levels.
```

Comment:
I think the file extension should be changed to `json5` before using comments.

Reply:
Removed the comments for now.